ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -225,16 +226,22 @@ export default class AlertTableRow extends React.Component {
           </FormGroup>
         </td>
         <td className="px-0">
-          <span
-            className={starred ? 'visible' : ''}
+          <Button
+            color="black"
```

Comment:
I have changed this component to reacstrap's Button. I had to insert `color="black"` to make it look visible, because of the automatically inserted `btn-secondary`.

Visually, what it changes is the `:focus`. This is the screenshot as it is in this PR:
![Screenshot of the current Bookmark button when focused, showing a blue outline around it](https://user-images.githubusercontent.com/3901809/70837803-175de580-1de4-11ea-82b4-df83deb91d8c.png)

There is no focus specific style as of it now.

Reply:
I think that's ok.